flocker/control/schema/endpoints.yml

```diff
@@ -182,6 +182,8 @@ definitions:
         path:
           '$ref': 'types.json#/definitions/node_path'
       required:
+      # FLOC-2160 - primary and path will no longer be required
+      # They will be ``None`` in non-manifest datasets
```

Comment:
Given comments above about how these values shouldn't be `None`, these should indeed not be required.